models/bert/__main__.py

```diff
@@ -63,9 +66,11 @@ def evaluate_split(model, processor, args, split='dev'):
 
     dataset_map = {
         'Reuters': ReutersProcessor,
-        'AAPD': AAPDProcessor,
         'IMDB': IMDBProcessor,
-        'SST-2': SST2Processor
+        'AAPD': AAPDProcessor,
+        'AGNews': AGNewsProcessor,
+        'Yelp2014': Yelp2014Processor,
+        'Sogou': SogouProcessor
```

Comment:
@Ashutosh-Adhikari SST-2 dataset is nw missing from the dataset_map. Can you please add it back?

Reply:
@achyudh sure.